src/bundle/Resources/public/js/scripts/sidebar/btn/user.edit.js

```diff
@@ -0,0 +1,16 @@
+(function (global, doc) {
+    const editButton = doc.querySelector('.ez-btn--edit-user');
+    const languageRadioOption = doc.querySelector('.ez-extra-actions--edit-user.ez-extra-actions--prevent-show [type="radio"]');
```

Comment:
I think you cannot assume there is only one language. After discussing with @webhdx and @Nattfarinn it should be possible to tranlsate user. You should keep in mind there might be more then one language option.

Reply:
There is only one language when there is `ez-extra-actions--prevent-show` class